_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1292 @@
+  [proposed][news366 lower feerate] adjusting the default minimum relay
+  feerate. The Bitcoin Core 29.1 release lowered the default minimum relay
+  feerate to 0.1 sat/vbyte in early September.
+
+{:#templatesharing}
+- **Peer block template sharing:** Anthony Towns proposed a
+  [way][news366 templ share] to improve the effectiveness of compact block
+  reconstruction in an environment where peers have divergent mempool policies.
+  The proposal would allow full nodes to send block templates to their peers,
+  which in turn would cache those transactions that would otherwise be rejected by
+  their mempool policies. The provided template contains transaction identifiers
+  encoded in the same format used by [compact block relay][topic compact block relay].
+
+  Later, in August, Towns opened [BIPs #1937][] to formally
+  [discuss the proposal][news368 templ share] for block template sharing.
+  During the discussion, several developers raised concerns about possible issues
+  related to privacy and node fingerprinting. In October, Towns decided
+  to [move the draft][news376 templ share] to the
+  [Bitcoin Inquisition Numbers and Names Authority][binana repo] (BINANA) repository
+  to address these considerations and to refine the document.
+  The draft was given the code [BIN-2025-0002][bin].
+
+{:#fuzzing}
+- **Differential fuzzing of Bitcoin and LN implementations:** Bruno Garcia
+  gave an update on the [progress and results][news369 fuzz] obtained by
+  [bitcoinfuzz][], a library to perform fuzz testing on Bitcoin and Lightning
+  implementations and libraries.
+  By leveraging the library, developers were able to report more
+  than 35 bugs in Bitcoin-related projects, such as btcd, rust-bitcoin,
+  rust-miniscript, LND, and more.
+
+  Garcia also highlighted the importance of differential fuzzing in the ecosystem.
+  Developers were able to report bugs on projects that do not implement fuzzing at
+  all, catch discrepancies between different Bitcoin implementations, and find gaps
+  in the Lightning specifications.
+
+  Finally, Garcia encouraged maintainers to integrate more projects into bitcoinfuzz,
+  expanding the support for differential fuzzing, and provided possible directions
+  for the future developments of the library.
+
+<div markdown="1" class="callout" id="stratumv2">
+
+## Summary 2025: Stratum V2
+
+[Stratum V2][topic pooled mining] is a mining protocol designed to replace the
+original Stratum protocol used between miners and mining pools. One of its key
+advantages is that it can allow individual pool members to choose which
+transactions to include in their blocks. This could improve Bitcoin's censorship
+resistance by distributing transaction selection across many independent miners.
+
+Throughout 2025, Bitcoin Core received several updates to better support Stratum
+V2 implementations. Some improvements earlier in the year were focused on
```

Comment:
```suggestion
Throughout 2025, Bitcoin Core received several updates to support Stratum
V2 implementations better. Some improvements earlier in the year were focused on
```

Reply:
I prefer the original phrasing, so leaving this as-is for now.